app/tests/test_views.py

```diff
@@ -299,6 +299,74 @@ def test_step_3_with_invalid_areaid_is_not_found(self):
         self.assert404(self.client.get('/my-expertise/blah'))
         self.assert404(self.client.get('/my-expertise/blah/1'))
 
+class MatchMeTests(ViewTestCase):
+    def setUp(self):
+        super(MatchMeTests, self).setUp()
+        self.login()
+
+        learn = LEVELS['LEVEL_I_WANT_TO_LEARN']['score']
+        refer = LEVELS['LEVEL_I_CAN_REFER']['score']
+        explain = LEVELS['LEVEL_I_CAN_EXPLAIN']['score']
+        do_it = LEVELS['LEVEL_I_CAN_DO_IT']['score']
```

Comment:
This was lifted from `test_user_match_against` and it's a lot easier to use than constantly referring to `LEVELS['some-really-long-string']['score']`. Might be useful to move the definitions into a common place or something.